### PR TITLE
Add getting and setting position to orbital camera.

### PR DESCRIPTION
**Pull request by sorskoot**
Add two methods to orbital camera:
- _getClosestPosition_. Gets a position and rotation (towards origin) based on a provided position on the orbit sphere of the camera.
- _setPosition_. Sets the position of the orbital camera on the sphere.

Fix issue where camera keeps moving if moving of mouse/touch stops.

### CODE REVIEW

File: orbital-camera.ts [hunk 1]
```diff
@@ -130,6 +135,62 @@ export class OrbitalCamera extends Component {
...
+        // Compute the azimuth angle (in radians)
+        const azimuth = Math.atan2(tempVec3[0], tempVec3[2]);
+        // Compute the polar angle (in radians)
+        const polar = Math.acos(tempVec3[1]);
+        const azimuthDeg = rad2deg(azimuth);
+        // Polar is inverted to match the orbital camera
+        const polarDeg = 90 - rad2deg(polar);
+        this._azimuth = azimuthDeg;
+        this._polar = polarDeg;
+        console.log(this._origin);
```
Comment (Squareys) [inline orbital-camera.ts:191]: Some debug log left over here 😌

File: orbital-camera.ts [hunk 2]
```diff
@@ -60,8 +64,11 @@ export class OrbitalCamera extends Component {
     private _azimuthSpeed: number = 0;
     private _polarSpeed: number = 0;
 
-    start(): void {
+    init() {
         this.object.getPositionWorld(this._origin);
```
Comment (Squareys) [inline orbital-camera.ts:68]: `start()` or maybe even `onActivate()` (to re-set the origin on reactivation) would probably make the most sense for this

Comment (sorskoot) [reply orbital-camera.ts:68]: It's a very deliberate choice to put it in `init`. If the component starts deactivated and the camera is moved the origin will be set to an incorrect value when the component is activated and `start` ran for the first time, if the origin is set at another moment that in `init`. This is exactly the scenario where I needed this component.

Comment (DavidPeicho) [reply orbital-camera.ts:68]: Using object and components in `init()` is forbidden though. It works but could break anytime. I guess we need to think about activation order @Squareys

File: orbital-camera.ts [hunk 3]
```diff
@@ -107,15 +114,13 @@ export class OrbitalCamera extends Component {
...
-        if (!this._mouseDown) {
-            /** Apply deceleration only when the user is not actively dragging */
-            this._azimuthSpeed *= this.damping;
-            this._polarSpeed *= this.damping;
-
-            /** Stop completely if the speed is very low to avoid endless tiny movements */
-            if (Math.abs(this._azimuthSpeed) < 0.01) this._azimuthSpeed = 0;
-            if (Math.abs(this._polarSpeed) < 0.01) this._polarSpeed = 0;
-        }
+        /** Always apply damping, because there's no event for stop moving */
```
Comment (DavidPeicho) [inline orbital-camera.ts:117]: `/*` for comment, here it's a documentation comment

File: orbital-camera.ts (hunk 2 above)
Comment (sorskoot) [reply orbital-camera.ts:68]: Any suggestions how this should work without having it in `init()`? Another component disables this one from its `start()` and `start()` on this component is never called until it is enabled.

**Pull request merged**